code/modules/clothing/head/helmet.dm

```diff
@@ -335,6 +335,89 @@
+
+/obj/item/clothing/head/helmet/monkey_sentience/equipped(mob/user, slot)
+	. = ..()
+	if(slot != ITEM_SLOT_HEAD)
+		return
+	if(!ismonkey(user) || user.ckey)
+		var/mob/living/something = user
+		to_chat(something, "<span class='boldnotice'>You feel a stabbing pain in the back of your head for a moment.</span>")
+		something.apply_damage(5,BRUTE,BODY_ZONE_HEAD,FALSE,FALSE,FALSE) //notably: no damage resist (it's in your helmet), no damage spread (it's in your helmet)
+		playsound(src, 'sound/machines/buzz-sigh.ogg', 30, TRUE)
+		return
+	magnification = user //this polls ghosts
+	visible_message("<span class='warning'>[src] powers up!</span>")
+	playsound(src, 'sound/machines/ping.ogg', 30, TRUE)
+	polling = TRUE
+	var/list/candidates = pollCandidatesForMob("Do you want to play as a mind magnified monkey?", ROLE_SENTIENCE, null, ROLE_SENTIENCE, 50, magnification, POLL_IGNORE_SENTIENCE_POTION)
+	polling = FALSE
+	if(!candidates.len)
+		magnification = null
+		visible_message("<span class='notice'>[src] falls silent. Maybe you should try again later?</span>")
+		playsound(src, 'sound/machines/buzz-sigh.ogg', 30, TRUE)
+	var/mob/picked = pick(candidates)
+	magnification.key = picked.key
+	playsound(src, 'sound/machines/microwave/microwave-end.ogg', 100, FALSE)
+	to_chat(magnification, "<span class='notice'>You're a mind magnified monkey! Protect your helmet with your life- if you lose it, your sentience goes with it!</span>")
+	icon_state = "[icon_state]up"
+
+/obj/item/clothing/head/helmet/monkey_sentience/Destroy()
+	. = ..()
+	disconnect()
+
+/obj/item/clothing/head/helmet/monkey_sentience/proc/disconnect()
+	if(!magnification) //not put on a viable head
+		return
+	if(!polling)//put on a viable head, but taken off after polling finished.
+		if(magnification.client)
+			to_chat(magnification, "<span class='userdanger'>You feel your flicker of sentience ripped away from you, as everything becomes dim...</span>")
+			magnification.ghostize(FALSE)
+		if(prob(10))
+			switch(rand(1,4))
+				if(1) //blood rage
+					magnification.aggressive = TRUE
+				if(2) //brain death
+					magnification.apply_damage(500,BRAIN,BODY_ZONE_HEAD,FALSE,FALSE,FALSE)
+				if(3) //primal gene (gorilla)
+					magnification.gorillize()
+				if(4) //genetic mass susceptibility (gib)
+					magnification.gib()
+	//either used up correctly or taken off before polling finished (punish this by destroying the helmet)
+	playsound(src, 'sound/machines/buzz-sigh.ogg', 30, TRUE)
+	playsound(src, "sparks", 100, TRUE)
+	visible_message("<span class='warning'>[src] fizzles and breaks apart!</span>")
+	magnification = null
+	new /obj/effect/decal/cleanable/ash/crematorium(src.loc) //just in case they're in a locker or other containers it needs to use crematorium ash, see the path itself for an explanation
+
+/obj/item/clothing/head/helmet/monkey_sentience/dropped(mob/user)
+	. = ..()
+	qdel(src)//runs disconnect code

```

Comment:
so if you take it out of the crate but then put it on a table, on the floor, in your backpack, etc., it just disintegrates on the spot?

Reply:
🤤 oh yea